packages/management/addons-manager/1.4.0/bundle/.imgpkg/images.yml

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
this file is not needed and can be removed

Reply:
removed images.yml